Readme.md

```diff
@@ -80,3 +80,24 @@ Note:
 1. We need `./app/client/font.js` available in the project with `fontfaceobserver` as a dev dependency
 2. If you have to compile any native addon (with `node-gyp`) make sure to install python,make and g++ in Docker file in build stage.
 
+### Compile static assets
+
+We support custom/static assets like fonts, icons, images, etc in order to compile it and build it separatly. For that you have to create a `static-assets` folder within `app`, Ex - `app/static-assets` and put all the custom assets within it. The below function will take care of rest of things like making a entry point for all the files and building it. 
+
```

Comment:
The code sample isn't needed. The user just needs to know which assets will be compiled (as in which directories/subdirectories will be picked). Also, could you pls change the wording a little? 
Something like this:

Static assets like fonts, icons, images, etc when placed in <insert dir name> will be bundled separately. This is done for convenience, since many frontend apps were doing this by themselves.